wd_core.au3

```diff
@@ -512,59 +512,59 @@ Func _WD_Window($sSession, $sCommand, $sOption = Default)
 	If $sOption = Default Then $sOption = ''
 
 	$sCommand = StringLower($sCommand)
-
+	Local $sURLSession = $_WD_BASE_URL & ":" & $_WD_PORT & "/session/" & $sSession

```

Comment:
it could be even:
```autoit
Local $sURLSession = $_WD_BASE_URL & ":" & $_WD_PORT & "/session/" & $sSession & "/" 
```

Reply:
Either way works for me, so I'll let you decide. 🙂